src/pcapi/scripts/booking/cancel_bookings_of_events_from_file.py

```diff
@@ -55,19 +57,23 @@ def _cancel_bookings_of_offers_from_rows(csv_rows: Iterable) -> None:
         try:
             repository.save(*bookings_to_cancel)
             logger.info(
-                f"[CANCEL BOOKINGS OF EVENTS FROM FILE] Annulation des réservations de l'offre '{offer_name}' d'id {offer_id} réussie"
+                "[CANCEL BOOKINGS OF EVENTS FROM FILE] Annulation des réservations de l'offre '%s' d'id %s réussie",
+                offer_name,
+                offer_id,
             )
             offers_successful.append(offer_id)
         except ApiErrors as error:
-            logger.exception(f"[CANCEL BOOKINGS OF EVENTS FROM FILE] {error.errors} pour l'offre d'id {offer_id}")
+            logger.exception(
+                "[CANCEL BOOKINGS OF EVENTS FROM FILE] %s pour l'offre d'id %s", str(error.errors), offer_id
+            )
             offers_in_error.append(offer_id)
 
-    logger.info(f"[CANCEL BOOKINGS OF EVENTS FROM FILE] {len(offers_successful)} OFFRES ANNULÉES")
-    logger.info(f"[CANCEL BOOKINGS OF EVENTS FROM FILE] LISTE DES OFFRES MISES À JOUR")
+    logger.info("[CANCEL BOOKINGS OF EVENTS FROM FILE] %s OFFRES ANNULÉES", str(len(offers_successful)))
```

Comment:
```suggestion
    logger.info("[CANCEL BOOKINGS OF EVENTS FROM FILE] %d OFFRES ANNULÉES", len(offers_successful))
```

... ou `%i`, que je découvre, ayant toujours utiilsé `%d`. Les habitudes, hein... :)